git-quick-stats: Add version 2.4.1

StarsbySea opened the pull request:
<!-- Provide a general summary of your changes in the title above -->

<!--
  By opening this PR you confirm that you have searched for similar issues/PRs here already.
  Failing to do so will most likely result in closing of this PR without any explanation.
  It is also mandatory to open a relevant issue (either Package Request or Bug Report) for
  discussion with the maintainers, before creating any new PR.
  Read the contributing guide first to save both your and our time.
-->

Closes #4226

![Install and Uninstall](https://user-images.githubusercontent.com/66008060/220019564-cfe73e29-381e-4a63-aa63-5b371c960faa.png)
![Run well](https://user-images.githubusercontent.com/66008060/220019573-47d1de9a-ba38-41a0-9cc1-e823c04dbfd0.png)

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

rashil2000 commented:
- You can just directly download the bash script from https://raw.githubusercontent.com/arzzen/git-quick-stats/2.4.1/git-quick-stats instead of the whole archive. 
- It's not a good idea to modify user's configuration files automatically. Instead, you can ask them to run that same command in the `notes` property
- Use `suggest` instead of `depends` for Git

StarsbySea commented:
* As this is considered one of the official installation methods, https://gist.github.com/arzzen/35e09866dfdadf2108b2420045739245, I don't really intend to modify it just to save a few MB of traffic. What if the script adds a new dependency?
* Modified as per comments
* Modified as per comments

StarsbySea commented:
/verify

rashil2000 commented:
> * As this is considered one of the official installation methods, https://gist.github.com/arzzen/35e09866dfdadf2108b2420045739245, I don't really intend to modify it just to save a few MB of traffic. What if the script adds a new dependency?

Highly unlikely. Tools written in Bash are always self-contained shell scripts. Besides we can always make a change if they do add a dependency.

StarsbySea commented:
/verify

rashil2000 commented on bucket/git-quick-stats.json:
```diff
@@ -0,0 +1,16 @@
+    "homepage": "https://git-quick-stats.sh/",
+    "license": "MIT",
+    "notes": "Run 'git config --global alias.quick-stats \"! $dir\\git-quick-stats\"' to configure alias.",
+    "suggest": "git",
```
```suggestion
    "suggest": {
        "Git": "git"
    },
```

StarsbySea commented:
/verify

rashil2000 commented:
Hmm, looks good to me. 

One last thing - we can add a `"bin": "git-quick-stats",` property too, so that people can also use it without needing to change the config.

StarsbySea commented:
/verify
